Dataset/CY_Pilot/consumer_reports_metadata.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "urn:ngsi-ld:Dataset:id:customer_reports:21675868",
+  "type": "Dataset",
+  "modified": "2024-04-09T14:09:32Z",
+  "source": [
+    "urn:ngsi-ld:Dataset:items:customer_reports:41266715"
+  ],
+  "description": [
+    "Reports by water consumers about issues they face related to their water. The dataset includes the reports, the assignment to operators and technical teams, as well as information about locations, type of problem, etc."
+  ],
+  "title": [
+    "Customer reports"
+  ],
+  "contactPoint": [
+    ""
+  ],
+  "belongsToCatalogue": "",
+  "distribution": [
+    ""
+  ],
+  "keyword": [
+    "water consumer",
+    "report",
+    "water issues",
+    "location",
+    "water problem type"
+  ],
+  "publisher": "WDME",
```

Comment:
Let's as it is but here we should put the details of the partner that provide this dataset